sound/soc/sof/intel/hda-stream.c

```diff
@@ -185,6 +185,17 @@ hda_dsp_stream_get(struct snd_sof_dev *sdev, int direction)
 			direction == SNDRV_PCM_STREAM_PLAYBACK ?
 			"playback" : "capture");
 
+	/*
+	 * Disable DMI Link L1 entry when there is capture stream is opened.
+	 * Workaround to address a known issue with host DMA that results
+	 * in xruns during pause/release in capture scenarios.
+	 */
+	if (IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_DISABLE_DMI_L1))
```

Comment:
that doesn't seem quite right to me.

This is not a debug option. You want DMI_L1 disabled by default, UNLESS the user chooses to enable DMI_L1 to e.g. look into power savings.

And this should be an Intel-specific Kconfig.

Reply:
@plbossart do you mean to say that we disable DMI L1 altogether by default irrespective of whether there is capture or not?

Reply:
@plbossart I've moved the config to intel Kconfig now. But Im not sure if we should disable DMI L1 altogether. The recommendation from Lech is to only disable it during capture.